be/src/runtime/exec_env.h

```diff
@@ -97,6 +97,11 @@ class ExecEnv {
     ClientCache<FrontendServiceClient>* frontend_client_cache() { return _frontend_client_cache; }
```

Comment:
Can be to remove frontend_client_cache(), broker_client_cache(), extdatasource_client_cache() in this place ?

Reply:
could not, because there are many users using this function. It need modify all references.